### DISCUSSION TITLE
Two CrazyFlies stuck on M2 solid light

### DISCUSSION THREAD

**Discussion by skovsen** [bitcraze/discussions#1239] [Q&A] [answered]
Hi.
I have two CFs that are now stuck with a solid M2 light, when I try to start them.
I have disconnected everything, but it persists.

I have followed the suggestions here: https://github.com/orgs/bitcraze/discussions/172
and I am able to get them in cold boot mode, as well as DFU update mode.
During that, I have tried to follow the bootloader recovery from this page: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/development/dfu/#automatic-dfu-flashing
Tried to flash new FW OTA using CFClient and tried to build my own and flashed it using DFU-util, but I end up with the same M2 solid light every time.

Is there anything else I can do?

**Comment by knmcguire** [comment]
There are several suggestions in that discussion you mentioned. Could you be more specific on which you tried exactly? Also which version of the Crazyflie client do you have?

**Reply by knmcguire** [reply]
Also take a look at this issue right here: https://github.com/bitcraze/crazyflie2-nrf-firmware/issues/95. Perhaps this is what is happening for you

**Reply by knmcguire** [reply]
And this discussion https://github.com/orgs/bitcraze/discussions/1206

**Reply by skovsen** [reply, marked as answer]
Hi. I've got them to connect again!
I'm not completely sure what made it work this time, as I think I just repeated my steps, but to give a usable for other having this problem here is what I did:
(Note: This is on MacOS 14.3.1)

1. Install Python 3.11.4
2. install cfclient 2024.3
3. Remove all decks from the CF
4. Start the CF in cold boot mode
5. From the cfclient connect in cold boot mode and upload firmware 2024.2
6. Let it upload and reboot and then I can now connect as usual

**Reply by knmcguire** [reply]
thanks for updating us! this is useful for others as well
